gobblin-service/src/main/java/org/apache/gobblin/service/modules/template/StaticFlowTemplate.java

```diff
@@ -67,25 +67,21 @@
   private transient Dag<JobTemplate> dag;
 
   private transient Config rawConfig;
-  private boolean isTemplateMaterialized;
 
-  public StaticFlowTemplate(URI uri, String version, String description, Config config,
+  public StaticFlowTemplate(URI flowTemplateDirUri, String version, String description, Config config,
```

Comment:
Is it possible to have Builder so that we can avoid so many parameters. It would be much earlier in the future to expand the functionality.